packages/blockchain-wallet-v4-frontend/src/data/auth/sagas.js

```diff
@@ -210,6 +215,8 @@ export default ({ api, coreSagas }) => {
       )
       // Redirect to error page instead of notification
       yield put(actions.alerts.displayError(C.WALLET_LOADING_ERROR))
+      // eslint-disable-next-line
+      console.log(e)
```

Comment:
use `console.error` and eslint wont complain

Reply:
btw console.error is also problematic. I just removed it, was using it for debugging